make/autoconf/platform.m4

```diff
@@ -597,6 +610,8 @@ AC_DEFUN([PLATFORM_SETUP_LEGACY_VARS_HELPER],
     OPENJDK_$1_OS_INCLUDE_SUBDIR="win32"
   elif test "x$OPENJDK_TARGET_OS" = "xmacosx"; then
     OPENJDK_$1_OS_INCLUDE_SUBDIR="darwin"
+  elif test "x$OPENJDK_TARGET_OS" = "xbsd"; then
```

Comment:
Do you really need this? This was introduced for consistency with legacy, where jni_md.h were supposed to reside in include/win32 or include/darwin, respectively. (In fact, since we never include anything but the jni_md.h file for the target platform, this could probably be removed completely now.)

If you skip this, the default assumption will be that the jni_md.h file will reside in include/bsd.